src/app/ReadHandler.h

```diff
@@ -188,6 +188,44 @@ class ReadHandler : public Messaging::ExchangeDelegate
     PriorityLevel GetCurrentPriority() const { return mCurrentPriority; }
     EventNumber & GetEventMin() { return mEventMin; }
 
+    enum class ReadHandlerFlags : uint8_t
+    {
+        // mHoldReport is used to prevent subscription data delivery while we are
```

Comment:
Here and for the other flag comments: they are still using the member names like `mHoldReport`....